static/js/composebox_typeahead.js

```diff
@@ -340,13 +336,13 @@ exports.compose_content_begins_typeahead = function (query) {
 
 exports.content_highlighter = function (item) {
     if (this.completing === 'emoji') {
-        return "<img class='emoji' src='" + item.emoji_url + "' /> " + item.emoji_name;
+        return typeahead_helper.render_typeahead_item(item.emoji_name, undefined, item.emoji_url);
```

Comment:
Yeah you're right. I think an even better solution would be to pass one object. What do you think?